mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -519,7 +683,7 @@
             <LayoutButtons index="9" Face="CloakOff" AbilCmd="GhostCloak,Off" Column="3"/>
             <LayoutButtons index="10" Face="EMP" AbilCmd="EMP,Execute" Column="1"/>
             <LayoutButtons index="11" Face="WeaponsFree" AbilCmd="GhostWeaponsFree,Execute" Row="1" Column="3"/>
-            <LayoutButtons index="12" Face="Cancel" AbilCmd="BypassArmorCancel,255" Row="2" Column="4"/>
+            <LayoutButtons index="12" Face="Cancel" AbilCmd="ChannelSnipe,Cancel" Row="2" Column="4"/>
```

Comment:
Ghost's Steady Targeting (Snipe) can now be manually canceled

This adds a cancel button and shortcut to the command card. There was a button definition before, but it was unused as the ability was not on the ghost anymore